var/spack/repos/builtin/packages/dd4hep/package.py

```diff
@@ -23,6 +23,7 @@ class Dd4hep(CMakePackage):
 
     tags = ['hep']
 
+    version('01-18', sha256='1e909a42b969dfd966224fa8ab1eca5aa05136baf3c00a140f2f6d812b497152')
```

Comment:
```suggestion
    version('1.18', sha256='1e909a42b969dfd966224fa8ab1eca5aa05136baf3c00a140f2f6d812b497152')
```
, and can you keep `master` on top? thanks!